tracking/translations_parser/cli/experiments.py

```diff
@@ -48,7 +64,16 @@ def parse_experiment(
     metrics = []
     if metrics_dir:
         for metrics_file in metrics_dir.glob("*.metrics"):
-            importer, dataset = metrics_file.stem.split("_", 1)
+            if mode == ExperimentMode.SNAKEMAKE:
+                # Snakemake experiments metrics are in the form `<importer><?_dataset>`
+                importer, *dataset = metrics_file.stem.split("_", 1)
+                dataset = dataset[0] if dataset else None
+            else:
+                # Taskcluster experiments metrics are in the form `<?augmentation_><dataset>`.
+                # As importer is a required attribute, we publish the metric base on its file
+                # name directly, making comparison with other type of publication impossible.
+                importer = metrics_file.stem
```

Comment:
@eu9ene That is were `importer` is missing. If there is some way to detect it (e.g. from its path) then I should be able to split augmentation and dataset (based on the underscore) and build a valid Metric object.

Example metric: `gs://moz-fx-translations-data--303e-prod-translations-data/
models/lt-en/opustrainer_no_student_aug_K1iHndFUSxSEDRLg_H9l1A/evaluation/student-finetuned/aug-upper-strict_wmt19.metrics`